tests/testthat/test-word.r

```diff
@@ -10,3 +10,9 @@ test_that("word extraction", {
 test_that("words past end return NA", {
   expect_equal(word("a b c", 4), NA_character_)
 })
+
+test_that("negative parameters", {
+  expect_equal("moon",word("walk the moon", -1, -1))
```

Comment:
Can you please add the missing space before `word()`?

Also you only need the first line here as the other two test the same code path.

Reply:
Done! 

Although I will suggest to keep the other two lines since they test a specific behavior: return all the sentence if the negative start parameter is greater than the number of words. Important to have a consistent API with `str_sub`.

Let me know if you want to remove it anyways!